inst/uvhydrograph/uvhydrograph.Rmd

```diff
@@ -22,6 +22,8 @@ cat(getLogo())
 ***
 
 ```{r echo=FALSE, warning=FALSE, message=FALSE, dev='svg', results='hide'}
+Sys.setenv(TZ=data$reportMetadata$timezone)
+
```

Comment:
I'm unsure if this is a good way to do it or if I should do this in the java layer (when setting up the Rserve request). I didn't want to do it for all reports so I put it in this specific report, but it can be tucked back into the java layer if we need to.